db/models/assessment_record/assessment_records.py

```diff
@@ -0,0 +1,48 @@
+from db import db
+from db.models.assessment_record.enums import Status
+from sqlalchemy import cast
+from sqlalchemy import Computed
+from sqlalchemy import func
+from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.dialects.postgresql import TEXT
+
+
+class AssessmentRecords(db.Model):
+    __tablename__ = "assessment_records"
+
+    application_id = db.Column(
```

Comment:
is there an arguemnt for this being uuid and the short-name being more hu8man readbale ?

Reply:
Changed to uuid.